content/post/abortion-bill.en.md

```diff
@@ -0,0 +1,83 @@
+
+This topic regarding messages content on social media is very interesting as U.S. law forces digital companies like Facebook to give access to their full users' data on request to governmental forces. And as we already know, social media knows their users better than their family or even themselves. So filtering or Big Data algorithms could be used by state police to flag and track people who underwent abortion outside of the state or worst, people who think (or are in the process of thinking) of doing so in the future. Worst science fiction scenarii in the manner of Inception movie are starting to become true. Regarding this, some naive people discover the extent of Google users’ spying when Google said recently that all their users' GPS waypoints that are related to an abortion center will be automatically erased from their servers[^8]. Meaning that Police could not claim unexisting (because erased) data. In addition, Google knows and memorizes a large history of geolocation data of their users, and not surprisingly, it also knows precisely to which building, company, trade or care center these coordinates correspond. To sum up, Google obviously knows from what address you start moving and what address is your destination, it knows what corresponds to these addresses (home, companies…). Maybe it was not a big deal when we thought that their business was to sell shoes and other goods, but their data reveals much more than your shopping. This is not new but this abortion scandal may have revealed it to more people.
+
+In addition, social media platforms may filter content related directly or indirectly to abortion in users’ states where abortion is not allowed. Some explanations, articles, documentaries on abortion may be censored just to respect the law… VPN services (secured connection that may be used to hide connection location) will have beautiful days ahead. The account of Abortion Finder, a site that allows women who wish to have an abortion to find a health professional, was briefly suspended from Instagram and then reinstated… Oops[^9].
+
+To improve their brand image, Meta promised to cover the costs of its employees having to travel outside their home state for an abortion, and Google claimed that its employees who live in states where abortion is banned may request to be transferred to other locations. Sadly, moving will be the only simple and safe way to deal with these freedom restrictions for women that are concerned and worried about it.
+
+If tech companies worries sincerely about their users' safety, they need to stop collecting our data now, and to not have it when the warrants arrive. At least they must collect less data, encrypt it, not share it with dubious actors, not force users to authenticate, etc. Of course this will negatively impact their business model but maybe it is time for them to change their retribution model. Society is evolving, they must also evolve if they have consideration for their customers.
```

Comment:
```suggestion
If tech companies worries sincerely about their users' safety, they need to stop collecting our data now, and to not have it when the warrants arrive. At least they must collect less data and at least encrypt them, not share it with dubious actors, force users to authenticate, and so on. Of course this will negatively impact their business model but maybe it is time for them to change their source of income. Society is evolving, they must also evolve if they have consideration for their customers.
```
Retribution is a really false friend here and have a much different meaning in English. Don't use it in that context.
Uncertain change: "not to have it" > "don't have them".

Reply:
I just changed the "retribution model"/"source of income" to "it" because it's basically talking about the business model.
The sentence is "they need to not have it", so no "do" here. Also data is singular.